.gitignore

```diff
@@ -102,3 +102,6 @@ dist
 
```

Comment:
You should probably add the views folder to your .ignore file as I believe that is what you are using for testing your apis.